packages/server/api/src/app/ai/mcp/mcp.ts

```diff
@@ -0,0 +1,56 @@
+import { logger } from '@openops/server-shared';
+import { ToolExecutionOptions } from 'ai';
+import { getMCPTool } from './mcp-tools';
+
+const MCP_TIMEOUT_MS = 10000; // 10 seconds
```

Comment:
should be an env var

Reply:
removed the file, it was not needed as the `ai` SDK does the tool calling now